sys/shell/commands/sc_netif.c

```diff
@@ -270,6 +270,10 @@ static void _netif_list(kernel_pid_t dev)
             printf("6LO  ");
         }
         linebreak = true;
+
+        printf("Link type: %s  ", (entry->flags & GNRC_IPV6_NETIF_FLAGS_IS_WIRED) ?
+                "wired" : "wireless");
+        linebreak= true;
```

Comment:
Just put it above https://github.com/RIOT-OS/RIOT/pull/3697/files#diff-ad63ecb056f596bac854115d8ccb46ebR300 e.g.